src/XenCert/StorageHandler.py

```diff
@@ -2399,15 +2399,17 @@ def FunctionalTests(self):
     def __del__(self):
         XenCertPrint("Reached StorageHandlerISCSI destructor")
         StorageHandler.__del__(self)
-        
+
+
 class StorageHandlerHBA(StorageHandler):
     def __init__(self, storage_conf):
         XenCertPrint("Reached StorageHandlerHBA constructor")
         StorageHandler.__init__(self, storage_conf)
+        self.sr_type = "lvmohba"
 
     def Create_SR(self):
```

Comment:
Create_SR() has been discarded. when you rebase Mark's code you will see he removed it.